modules/guAnalyticsAdapter.js

```diff
@@ -163,10 +165,12 @@ function trackAuctionEnd(args) {
   return [event];
 }
 
+// Protect against setting undefined or null values
 function setSafely(obj, key, value) {
-  if (value) {
-    Object.assign(obj, {[key]: value});
+  if (value === undefined || value === null) {
+    return;
   }
+  Object.assign(obj, {[key]: value});
```

Comment:
Does this work?  Or do you have to explicitly `return`?

Reply:
That's fine, because we don't care about any return value. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions . The default is undefined.